cli/src/main/java/hudson/cli/CLI.java

```diff
@@ -485,10 +485,8 @@ private static String computeVersion() {
         try {
             InputStream is = CLI.class.getResourceAsStream("/jenkins/cli/jenkins-cli-version.properties");
             if (is != null) {
-                try {
+                try (is) {
                     props.load(is);
-                } finally {
-                    is.close();
                 }
             }
```

Comment:
try-with-resources is `null` safe.

Why not

```
        try (InputStream is = CLI.class.getResourceAsStream("/jenkins/cli/jenkins-cli-version.properties")) {
            if (is != null) {
                    props.load(is);
                }
            }
```

Reply:
Pre-existing issue